docs/source/contributing/setup.rst

```diff
@@ -87,6 +89,7 @@ happen.
    The second line needs to be run every time you open a new terminal.
 
 4. Install the python packages required for JupyterHub development.
+   Question: If miniconda is the recommended way to get python should these installation instructions be updated to use ``conda`` instead of ``pip``?
```

Comment:
I'd leave this as pip. Using conda to get Python doesn't preclude using pip. Conda 4.6 and later behaves pretty well with pip as a peer.